src/librustdoc/doctest.rs

```diff
@@ -414,6 +430,9 @@ pub fn make_test(
             let sm = Lrc::new(SourceMap::new(FilePathMapping::empty()));
             let emitter =
                 EmitterWriter::new(box io::sink(), None, false, false, false, None, false);
+
+            supports_color = emitter.supports_color();
```

Comment:
Why is this calling `supports_color` on this `Emitter`? Surely this will always return `false` because of how it's constructed above?

Reply:
Indeed, just realized it afterwards. I'll update it.